admin/src/lib.rs

```diff
@@ -0,0 +1,323 @@
+// Copyright 2021-2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Experimental admin functionality for parity-db.
+
+// Note standard substrate db uses:
+// NUM_COLUMN 12
+// Column 1 (state):
+// 
+//		state_col.ref_counted = true;
+//		state_col.preimage = true;
+//		state_col.uniform = true;
+//
+// TODO Passing parameter to define this is be a bit tedious,
+// using a existing metadata file could be good.
```

Comment:
What exactly does this TODO suggest? Note that column settings for substrate are likely to change soon. And it should not matter for this tool anyway.

Reply:
I had previously everything as parameter, so when I wanted to change some metadata I had to redefine all compression for all columns.
So the suggestion was to write a reading metadata method to update it from parameter (which is in you prev pr).
But now we use a dest_metadata so this is totally out of date.